src/components/inputs/InputAmount/index.tsx

```diff
@@ -144,7 +155,17 @@ function InputAmount(
         <InputContainer>
           <Side>
             {selectedToken?.id && tokens.length < 2 && (
-              <CoinIcon name={selectedToken.id} size={CoinSizes.SMALL} coin={selectedToken.id} />
+              <>
+                {customCoinIcon ? (
+                  <>{renderCustomIcon}</>
```

Comment:
I think we can replace it by simply `{customCoinIcon}` since you already check if it's passed.